src/pip/_internal/cli/req_command.py

```diff
@@ -58,7 +58,7 @@ def _create_truststore_ssl_context() -> Optional["SSLContext"]:
         return None
 
     try:
-        import truststore
+        from ..._vendor import truststore
     except ImportError:
         raise CommandError(
             "To use the truststore feature, 'truststore' must be installed into "
```

Comment:
This error message needs to be updated.

Reply:
I also wonder if it’d be a good idea to add an explicit `sys.version_info` check before we try to import the package so the slowdown on lower versions would be less significant.

Reply:
Great catch, I'll push a change that forwards the import error message.

Reply:
> I also wonder if it’d be a good idea to add an explicit `sys.version_info` check before we try to import the package so the slowdown on lower versions would be less significant.

I believe that's already done, we check that `sys.version_info >= (3, 10)` immediately above.

Reply:
Updated the error message: https://github.com/pypa/pip/pull/12107/commits/a1c537dcd83c44c66973d2cced8ab3ec5120f2fd